Services/DataX.Utilities.JobRunner/DataX.Utilities.JobRunner/StorageTableBase.cs

```diff
@@ -0,0 +1,81 @@
+﻿// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+
+using DataX.Contract;
+using Microsoft.Azure.Cosmos.Table;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace DataX.Utilities
+{
+    /// <summary>
+    /// Utility base class for CosmosTable

```

Comment:
Is this CosmosDB?

Reply:
This is CosmosDB Table. The traditional CosmosDB that we use for Data Accelerator has Api=-Core (sql) where as this has api=AzureTable, something we set at the time of creation of the resourceL cosmosDB.